EmbeddingTestBeds/Embedding.Droid/Embedding.Droid.csproj

```diff
@@ -140,32 +109,12 @@
   <ItemGroup>
     <AndroidEnvironment Include="environment.txt" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Xamarin.Android.Support.v4">
+      <Version>27.0.2</Version>
+    </PackageReference>
+  </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-  <Import Project="..\..\packages\Xamarin.Android.Support.Annotations.27.0.2\build\MonoAndroid81\Xamarin.Android.Support.Annotations.targets" Condition="Exists('..\..\packages\Xamarin.Android.Support.Annotations.27.0.2\build\MonoAndroid81\Xamarin.Android.Support.Annotations.targets')" />
```

Comment:
The weird thing about putting `.g.targets` files into the `obj` directory is that VS caches project import state. This is why its necessary to unload/reload projects to edit the project file -- VS needs to refresh it's cache. But no such provisions are made for the obj dirs. Everyone assumes they can be erase and reconstituted without having to reload the project... So why why add those `.g.targets` files in a more durable location. After all, the nuget package they're supporting is immutable so why re-generate them? Why not cache them with the restored nuget package?

Reply:
https://docs.microsoft.com/en-us/visualstudio/msbuild/customize-your-build?view=vs-2017#msbuildprojectextensionspath

There's the magic. 

And nuget now has a dep on msbuild because only msbuild can figure out where the output directory is (or, more exactly, the value of `MSBuildProjectExtensionsPath`). Which in turn means that nuget, previously decoupled from msbuild, now has a dep on the entire graph of target/proj files all just so they can figure out where to store the generated targets. Seems a wildly large hammer. Msbuild already takes a dep on figuring out where the nuget lib dir is. Why not have it also dep on where the generated target files live? Then nuget wouldn't need to dep on msbuild... well, I'm sure they had a good reason.